desktop-app/app/components/WebView/screenshotUtil.js

```diff
@@ -53,6 +54,23 @@ export const captureFullPage = async (
     viewPortWidth,
   } = await webView.executeJavaScript(`
     document.body.classList.add('responsivelyApp__ScreenshotInProgress');
+
+    // add shadow div to make it fit with totalHeight
```

Comment:
@manojVivek i added it that is caused more white space on scroll capture. why i need to add it. maybe this image could explain this code
![Untitled Diagram](https://user-images.githubusercontent.com/11748183/86424268-4a2de380-bd0c-11ea-9d24-e70a5087fdf8.png)

Reply:
@sprabowo But the screenshot logic already handles the first case properly, if I'm not wrong.

It is only those off the screen `webview`'s that were creating the problem. 
Can you please try once with only the zoom-out change and without this additional height change?